### DISCUSSION TITLE
Install and setup tagalong

### DISCUSSION THREAD

**Discussion by zekhoi** [NvChad/NvChad#1538] [Q&A]
Hi, i am new to vim.
Can someone teach me how to install and setup tagalong(https://github.com/AndrewRadev/tagalong.vim)? 
I already install it with packer but it doesnt work because i dont know how to setup the tagalong in NvChad.

![image](https://user-images.githubusercontent.com/55708473/188577283-91294a5a-0d84-4a75-aa54-c3ea0c443da9.png)

**Comment by siduck** [comment]
what did you do till now
